test/source/patterns.ts

```diff
@@ -98,9 +99,51 @@ for (const buildType of ['chrome-consumer', 'chrome-enterprise']) {
       }
     }
   }
-  const gmailCs = manifest.content_scripts?.find(cs => cs.matches?.includes('https://mail.google.com/*'));
-  if (!gmailCs?.css?.length || !gmailCs.js?.length) {
-    console.error(`Missing content_scripts declaration for Gmail in ${buildType}/manifest.json`);
+  if (buildType === 'thunderbird-consumer') {
+    if (manifest.manifest_version !== 2) {
+      console.error(`${buildType} - The manifest version is not 2`);
+      errsFound++;
+    }
+    if (!Array.isArray(manifest.web_accessible_resources)) {
+      console.error(`${buildType} - The web_accessible_resources should be an array`);
+      errsFound++;
+    }
+    if (typeof manifest.content_security_policy !== 'string') {
+      console.error(`${buildType} - The content_security_policy should be a string`);
+      errsFound++;
+    }
+    const expectedHostPermissions = [
+      'alarms',
+      'storage',
+      'tabs',
+      'scripting',
+      'unlimitedStorage',
+      'identity',
+      'compose',
+      'messagesRead',
+      'messagesUpdate',
+      'messagesModify',
+      'accountsRead',
+      'https://flowcrypt.com/*',
+      'https://*.google.com/*',
+      'https://outlook.live.com/*',
+      'https://outlook.office365.com/*',
+      'https://outlook.office.com/*',
+      'https://graph.microsoft.com/*',
+      'https://login.microsoftonline.com/*',
+      'https://www.googleapis.com/*',
+    ];
```

Comment:
we already checked `expectedPermissions` and `expectedHostPermissions` earlier, here let's check only for Thunderbird specific permissions (`'messagesRead', 'messagesUpdate', 'messagesModify', 'accountsRead'`)